test/NuGet.Server.Core.Tests/ServerPackageRepositoryTest.cs

```diff
@@ -365,9 +394,9 @@ public void ServerPackageRepositoryReadsDerivedData()
 
                 // Act
                 var packages = serverRepository.GetPackages();
-                var singlePackage = packages.Single() as ServerPackage;
 
                 // Assert
+                var singlePackage = packages.Single();
```

Comment:
I think this should be `SingleOrDefault()` instead of `Single()`.  (See next statement.)